group1/comm/templates/comm/index.html

```diff
@@ -96,14 +97,15 @@ <h4 class="sidebar_description">Quick links</h4>
             <button id="upload file" class="btn btn-default" type="button" data-toggle="tooltip" title="Send files!" onclick="filechoose()">
               <span class="glyphicon glyphicon-folder-open" aria-hidden="true"></span>
             </button>
-            <button id="sendcleartext" class="btn btn-default" type="button" data-toggle="tooltip" title="Start Video Share!" onclick="window.location.href='/communication/videochat'">
+            <button id="startVideo" class="btn btn-default" type="button" data-toggle="tooltip" title="Start Video Share!" onclick="window.location.href='/communication/videochat'">
             <!-- 
             temporarily removed in order to fix a merge conflict; 10/29 DB
             <button id="sendcleartext" class="btn btn-default" type="button" data-toggle="tooltip" title="Start Video Share!" onclick="startVideoChat()"> 
             -->
                 <span class="glyphicon glyphicon-facetime-video" aria-hidden="true"></span>
             </button>
-            <div class="btn-group btn-group-inline">
+            
+	    <div class="btn-group btn-group-inline">
```

Comment:
It looks like the spacing here might be slight off (1 or 2 spaces?).

Reply:
fixed in 
c61e863